features/api-resource-mgt/org.wso2.carbon.identity.api.resource.mgt.server.feature/resources/api-resource-collection.xml.j2

```diff
@@ -1237,6 +1237,17 @@
             </Read>
         </Scopes>
     </APIResourceCollection>
+    <APIResourceCollection name="oidcScopes" displayName="OIDC Scopes" type="organization">
+            <Scopes>
+                <Feature>
+                    <Scope name="console:org:scopes:oidc"/>
+                    <Scope name="console:org:scopes:oidc_view"/>
+                </Feature>
+                <Read>
+                    <Scope name="internal_org_oidc_scope_mgt_view"/>
+                </Read>
+            </Scopes>
+        </APIResourceCollection>
```

Comment:
Shalll we fix formatting?

Reply:
Fixed with [6544946](https://github.com/wso2/carbon-identity-framework/pull/7125/commits/6544946f6fa3457bc89827d7a1380ba5743ea24b).